webclient/components/1time-Mrow-1Field/service-statement/db/client-side/initialize-on-first-load.vue

```diff
@@ -8,12 +8,26 @@
 <script>
 import initializeSsComponent from '@/components/1time-Mrow-1Field/service-statement/db/client-side/static-data/insert-into-master-of-search-phrases-ct.vue'
 import insertIntoMasterTblOfServiceStatementDuringInitOfCt from '@/components/1time-Mrow-1Field/service-statement/db/client-side/static-data/insert-into-master-of-service-statement-ct.vue'
+import clientSideTable from '~/components/1time-Mrow-1Field/service-statement/db/client-side/structure/patient-table-of-service-statements.js'
```

Comment:
more readable would be:

step1-init-search-phrases
step2-init-ss-field-names
step3-get-latest-data-from-db-server